build.gradle

```diff
@@ -222,6 +222,9 @@ dependencies {
   implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.2.2.RELEASE'
   implementation group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.0.4'
   implementation group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.1.2'
+
+  implementation group: 'org.elasticsearch', name: 'elasticsearch', version: '7.11.1'
```

Comment:
I think we can use just java client lib:
`  implementation group: 'org.elasticsearch.client', name: 'elasticsearch-rest-high-level-client', version: '7.12.1'`

Reply:
i replaced the library by the client one